app/serializers/api/applications_csv_serializer.rb

```diff
@@ -96,5 +96,12 @@ def updated_at(application)
         application.updated_at,
       ].compact.max.rfc3339
     end
+
+    def created_at(application)
```

Comment:
same here regarding keeping this the same, its only on participant serialisers that we need to change this?